libnmstate/nm/team.py

```diff
@@ -16,7 +16,7 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
-
+import copy
```

Comment:
An additional space between copy and the LICENSE comment would be good but not a must, just a nit. Thanks.